### PR TITLE
Added "Artificial Intelligence" and "Cybersecurity" to the typed words.

### PR DESCRIPTION
**Pull request by Laan33**
Added "Artificial Intelligence" and "Cybersecurity" to the typed words.

Set the typing to finish on "Computer".
Further info - we have not enabled looping, this typing will go through the list, and then finish on "Computer Society".

Screenshot 2024-11-13 at 01 10 28
Screenshot 2024-11-13 at 01 10 23

### CODE REVIEW

**Comment by Laan33** [comment]
I believe @Daxorinator is very passionate on this addition, so I would ask for @Daxorinator's review before merging.

File: src/components/splash/splash.component.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ const HeroComponent = () => {
 
     React.useEffect(() => {
         const typed = new Typed(el.current, {
-            strings: ["Computer", "Developer", "Technology"], // Strings to display
+            strings: ["Computer", "Developer", "Technology", "Artificial Intelligence", "Cybersecurity", "Computer"], // Strings to display
```
Comment (Daxorinator) [inline src/components/splash/splash.component.tsx:10]: Computer listed twice? Does this loop, or does it stop at the end?

**Review by Daxorinator — CHANGES_REQUESTED** [review]

File: src/components/splash/splash.component.tsx (hunk 1 above)
Comment (Laan33) [reply src/components/splash/splash.component.tsx:10]: it stops on computer, see the description 🙏

Comment (Daxorinator) [reply src/components/splash/splash.component.tsx:10]: Resolved

**Review by Daxorinator — APPROVED** [review]

**Pull request merged**